app/src/main/java/org/schabi/newpipe/local/history/HistoryRecordManager.java

```diff
@@ -244,9 +244,9 @@ private boolean isStreamHistoryEnabled() {
                 .subscribeOn(Schedulers.io());
     }
 
-    public Flowable<List<SearchHistoryEntry>> getRelatedSearches(final String query,
-                                                                 final int similarQueryLimit,
-                                                                 final int uniqueQueryLimit) {
+    public Flowable<List<String>> getRelatedSearches(final String query,
+                                                     final int similarQueryLimit,
+                                                     final int uniqueQueryLimit) {
```

Comment:
Do you want to try adding a test for this? Similiar to `LocalPlaylistManagerTest.kt`

Reply:
Done, and I also found a bug, thank you for reminding me to write tests :-)